templates/422/invalid_huc.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html' %}
+{% block content %}
+<h3>Invalid HUC-8</h3>
+<p>
+  The HUC-8 code you provided is not supported.
```

Comment:
Switch "is not supported" to "is invalid".  We're using the single "truth" list of HUC8, if someone provides a code that we don't have, it's invalid.

Reply:
The text has been changed to say "invalid" in c1b774a.